pyaedt/application/Variables.py

```diff
@@ -945,8 +950,10 @@ def delete_variable(self, sVarName):
 
 class Variable(object):
     """Variable class.
```

Comment:
```suggestion
    """Stores variables and provides for performing operations on variables.
```
Let's write the short descriptions like this from now on.